doc/source/examples/incompressible-flow/3d-taylor-green-vortex/3d-taylor-green-vortex.rst

```diff
@@ -0,0 +1,374 @@
+
+
+----------------------
+Results and Discussion
+----------------------
+
+The flow patterns generated by the Taylor-Green vortex are quite complex. The following animation displays the evolution of velcoity iso-contours as the vortex break downs and generate smaller structures.
+
++----------------------------------------------------------------------------------------------------------------------------------------------------+
+| .. raw:: html                                                                                                                                      |
+|                                                                                                                                                    |
+|    <iframe width="520" height="400" src="https://www.youtube.com/embed/GGij2g_yz5g?si=Q81lcHyPd9ONxqT4"  frameborder="0" allowfullscreen></iframe> |
+|                                                                                                                                                    |
++----------------------------------------------------------------------------------------------------------------------------------------------------+
+
+
+Using the ``enstrophy.dat`` and ``kinetic_energy.dat`` files generated by Lethe, the temporal decay of the kinetic energy can be monitor. First, we calculate the time-derivative of the kinetic energy by invoking the script present in the example folder:
+
+.. code-block:: text
+  :class: copy-button
+
+  python3 calculate_dissipation_rate.py -i kinetic_energy.dat -o output.dat
+
+Then, by invoking the script present in the example, a plot compairing the kinetic energy decay with the enstrophy is generated:
+
+.. code-block:: text
+  :class: copy-button
+
+  python3 plot_dissipation_rate.py -ke kinetic_energy_decay.dat -ens enstrophy.dat -v 0.000625
+
+.. note:
+ 
+  A nice plot with a zoomed in section can be generated by adding the argument `-z True` to the command above.
+
+The following plot shows the decay of kinetic energy as measured
+
++-------------------------------------------------------------------------------------------------------------------+
+|  .. figure:: images/dissipation_comparison_Q2_32.png                                                              |
+|     :width: 500                                                                                                   |
+|     :alt: In this figure, the kinetic energy decay is compared to the enstrophy for a 32^3 Q2Q2 mesh.             |
+|           The two curves should be identical in the absence of numerical dissipation                              |                         
```

Comment:
This descriptions are not really appearing anywhere... they just add a line under the figure. Maybe we can remove them?

Reply:
I think it's a bit overkill. It was a lot of effort to add that and align the table if it's only going to be used when figures don't display, I'll remove it.